### PR TITLE
add tests and build for python 3.9

### PR DESCRIPTION
**Pull request by samuelcolvin**
replace #15.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -127,12 +127,10 @@ jobs:
         override: true
 
     - name: install python dependencies
-      run: |
-        pip install -U setuptools wheel
-        pip install 'git+https://github.com/joerick/cibuildwheel.git@refs/pull/194/head'
```
Comment (homeworkprod) [inline .github/workflows/ci.yml:129]: Yeah, I was looking into this as well. That PR has been merged in March 2020. The CI says it used cibuildwheel v1.2.0 with v1.8.0 being the most recent release.

**Comment by homeworkprod** [comment]
Looking good so far!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (samuelcolvin) [reply .github/workflows/ci.yml:129]: this was a temporary hack before cibuildwheel master/release versions support github actions. Now looks to working well with the latest release.

**Comment by samuelcolvin** [comment]
I'll make a new release now.

**Comment by homeworkprod** [comment]
Excellent, thanks a lot.

**Pull request merged**